src/runtime-pmix.c

```diff
@@ -48,7 +49,7 @@ shmem_runtime_init(void)
         return rc;
     }
 
-    (void)strncpy(proc.nspace, myproc.nspace, PMIX_MAX_NSLEN);
+    (void)strncpy(proc.nspace, myproc.nspace, PMIX_MAX_NSLEN+1);
```

Comment:
Yeah, this `+1` does kinda make sense to me - it's for the NULL terminator, correct?

Might want to see if @rhc54 has any thoughts.  If I grep the PMIx examples with:
```
grep -r "strncpy.*nspace.*PMIX_MAX_NSLEN*" * 
```
I see a lot of occurrences, but none with the `+1`.  Does it matter?

Reply:
It would be safer to use the``` PMIX_LOAD_NSPACE``` macro to ensure everything is correct (i.e., fits in space and is NULL terminated). The ```+1``` is for the latter purpose - if we tell people "here is the max name length", then we have to ensure there is one more spot for the NULL terminator.

Reply:
Thanks @rhc54.  I don't see `PMIX_LOAD_NSPACE` in the 3.0 spec.  Is it new?

Reply:
Ah, indeed - it is in the 3.1 spec. It was overlooked when we wrote the 3.0 version and so was considered a correction as opposed to an addition. See https://pmix.org/wp-content/uploads/2019/01/pmix-standard-3.1.pdf for the updated spec.

Reply:
Updated to use `PMIX_LOAD_NSPACE` in https://github.com/Sandia-OpenSHMEM/SOS/pull/837/commits/3b11877e256e3fa3613abb06ba516ce3c8b3a66b.